lib/inferno/dsl/fhir_evaluator/rule.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module FhirEvaluator
+  class Rule
+    def check(_context)
+      raise 'not implemented'
+    end
+
+    # Get the subclasses of this class, ie, the actual implemented Rules.
+    def self.descendants
```

Comment:
I think `Class.descendants` already exists and you shouldn't have to reimplement it.

Reply:
Removed

Reply:
You don't need the `inherited` hook either.

Reply:
Removed